### ISSUE TITLE
Develop milestones to have a roadmap for this initiative

### ISSUE THREAD

**Issue by niconoe** [tdwg/camtrap-dp#31]
In GitLab by @kbubnicki on Aug 27, 2020, 4:10

I believe we need a well defined milestones for this initiative especially that I expect that more and more people will join us soon. So the steps could be e.g.

1. Develop JSON schemas in a relatively small group of people.
2. Consult developed schemas with a broader audience of camera trapping researchers and practitioners (online questionnaires?)
3. Develop R/Python tools to facilitate work with camtrap data packages (e.g. camtrapR)
4. Write a paper.
5. Integrate the solution with the existing camtrap data management software (Agouti, TRAPPER). Actually these developments are partly ongoing and can be done in parallel to the other points.

This should be described in more details and preferably some deadlines for each step should be also specified.

**Comment by niconoe** [comment]
In GitLab by @kbubnicki on Aug 27, 2020, 16:12

We can use gitlab milestones functionality: https://gitlab.com/oscf/camtrap-package-schemas/-/milestones

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Aug 28, 2020, 11:42

Thanks for summarizing this!

- I'm currently working on 1 and 5. I guess we want to tackle most of the open issues before starting 2? I think we need a hard deadline for this, so 2 does not get postponed.
- 2 can be done in the framework of Alter-NET (@milotictanja?) and TDWG. For starters we need a better README if we want to point people here.
- 3 we can already explore what should be done, before actually implementing it
- 4 this is a good idea, I'm happy to collaborate

**Comment by niconoe** [comment]
In GitLab by @milotictanja on Aug 28, 2020, 12:32

I totally agree with @peterdesmet : for step 2 a consultation of the Alter-NET partners in the multi-site research project would be great as we planned similar actions in the framework of the workshop that was cancelled. I also agree that in that stage a better README is indispensable because many of these project partners are scientists with (often) limited technical skills.

**Comment by niconoe** [comment]
In GitLab by @kbubnicki on Aug 28, 2020, 12:54

Perfect! I would also use other alternative channels to reach to even broader audience e.g. https://www.wildlabs.net/ etc

**Comment by niconoe** [comment]
In GitLab by @kbubnicki on Aug 28, 2020, 12:56

Also would be good the get some feedback from people behind https://www.wildlifeinsights.org

**Comment by niconoe** [comment]
In GitLab by @peterdesmet on Sep 1, 2020, 11:42

changed the description
